test/index.test.js

```diff
@@ -73,7 +73,7 @@ it('Should throw when trying to register multiple instances without giving a nam
   }
 })
 
-it('Expose a getOrama function', async () => {
+it('Expose a withOrama function', async () => {
```

Comment:
good naming - could you add a note in the readme?

Reply:
Added a new section in the Readme 🚀